src/swift_server.rs

```diff
@@ -185,6 +195,108 @@ pub async fn process_order(
                 }),
             );
         }
+    };
+
+    if user.is_none() {
+        log::warn!(
+            target: "server",
+            "{log_prefix}: Error simulating order, user not found"
+        );
+        return (
+            axum::http::StatusCode::BAD_REQUEST,
+            Json(ProcessOrderResponse {
+                message: PROCESS_ORDER_RESPONSE_ERROR_USER_NOT_FOUND,
+                error: Some("user not found".to_string()),
+            }),
+        );
+    }
+    let user = user.unwrap();
+
+    // If fat fingered order that requires sanitization, then just send the order
+    let mut order_params = order_params.clone();
+    if server_params
+        .simulate_will_auction_params_sanitize(&mut order_params)
+        .await
+    {
+        let uuid = String::from_utf8(uuid.to_vec()).expect("invalid utf8 uuid");
```

Comment:
don't need the intermediate Vec
```suggestion
        let uuid = std::str::from_utf8(&uuid).expect("invalid utf8 uuid").to_string();
```